src/OctoshiftCLI.Tests/gei/Handlers/GenerateScriptCommandHandlerTests.cs

```diff
@@ -1408,7 +1409,7 @@ public async Task Sequential_Ghes_Single_Repo_Aws_S3()
                 .Setup(m => m.GetRepos(SOURCE_ORG))
                 .ReturnsAsync(new[] { REPO });
 
-            var expected = $"Exec {{ gh gei migrate-repo --github-source-org \"{SOURCE_ORG}\" --source-repo \"{REPO}\" --github-target-org \"{TARGET_ORG}\" --target-repo \"{REPO}\" --ghes-api-url \"{ghesApiUrl}\" --aws-bucket-name \"{AWS_BUCKET_NAME}\" --wait }}";
+            var expected = $"Exec {{ gh gei migrate-repo --github-source-org \"{SOURCE_ORG}\" --source-repo \"{REPO}\" --github-target-org \"{TARGET_ORG}\" --target-repo \"{REPO}\" --ghes-api-url \"{ghesApiUrl}\" --aws-bucket-name \"{AWS_BUCKET_NAME}\" --aws-region \"{AWS_REGION}\" --wait }}";
```

Comment:
Should we have a test which covers what happens if you don't submit a region? (I think we said that the region would be required for `generate-script`, but not `migrate-repo`.)

Reply:
see [this](https://github.com/github/gh-gei/pull/844#discussion_r1123632495)